packages/salesforcedx-utils-vscode/src/providers/configAggregatorProvider.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { ConfigAggregator } from '@salesforce/core';
-import { workspaceUtils } from '../util';
+import { workspaceUtils } from '../workspaces';
```

Comment:
So the way we have our directory structure across packages is kind of wacky right now.  The workspaceUtils seems like it would live in the workspace dir so I moved it there.